product-page/product.js

```diff
@@ -0,0 +1,27 @@
+const body = document.querySelector("body");
+const btnFilter = document.querySelector("#filter-btn");
+const btnSort = document.querySelector("#sort-btn");
+const filterScreen = document.querySelector(".filter-layout-mob");
+const sortScreen = document.querySelector(".sort-layout-mob");
+
```

Comment:
Hey, i see you have added some js here, was it supposed to be functional? Because the filter and sort button in mobile view are not working as of now.

Reply:
I checked into it. The code sandbox was not taking the javascript code. I have made another sandbox link. Can you check here.
CSB link(updated): https://h5rt7.csb.app/product.html

Reply:
Yes it's working fine now.👍